projects/backend/fronts.ts

```diff
@@ -322,6 +341,11 @@ export const transformToFront = async (
     }
 }
 
+// Run through each card and reduce or remove trailImage as appropriate
+const optimisedFrontImages = (front: Front): Front => {
```

Comment:
Is this waiting to be implemented in another PR?

Reply:
Oh. No, that's left over from a abandoned tack. Thanks for spotting, will remove.